node/network/bridge/src/action.rs

```diff
@@ -0,0 +1,184 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
```

Comment:
I know it's hard to believe 2020 is over :)
```suggestion
// Copyright 2020-2021 Parity Technologies (UK) Ltd.
```